bedrock/firefox/views.py

```diff
@@ -515,6 +522,21 @@ def get_template_names(self):
                 template = "firefox/developer/whatsnew.html"
             else:
                 template = "firefox/whatsnew/index.html"
+        elif version.startswith("119."):
+            if locale in ["de", "fr", "en-GB"] or country == "GB":
+                if variant == "3":
+                    template = "firefox/whatsnew/whatsnew-fx119-eu-relay.html"
+                elif variant == "2":
+                    template = "firefox/whatsnew/whatsnew-fx119-eu-trio-v2.html"
+                else:
+                    template = "firefox/whatsnew/whatsnew-fx119-eu-trio-v1.html"
```

Comment:
Oh, these two templates also seem to be missing the experiment JS, so the Traffic Cop redirect isn't triggering got the EU versions?

Reply:
it's in trio-v1 and trio-v2 just extends that template so it should appear there as well... or do I need to repeat the block in v2?

Reply:
Oh so it is, then how come I didn't get redirected. Hmm, let me try again

Reply:
Ah, never mind - I tested in a private window and that had DNT enabled. Seems to work fine 👍